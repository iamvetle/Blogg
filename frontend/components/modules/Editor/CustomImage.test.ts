import { Image }from './CustomImage'

test('Customimage should match snapshot', () => {
    expect(Image).toMatchSnapshot()     
})