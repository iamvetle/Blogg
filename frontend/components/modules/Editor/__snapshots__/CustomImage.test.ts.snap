// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Customimage should match snapshot 1`] = `
Node {
  "child": null,
  "config": {
    "addAttributes": [Function],
    "addCommands": [Function],
    "addInputRules": [Function],
    "addOptions": [Function],
    "defaultOptions": {},
    "draggable": true,
    "group": [Function],
    "inline": [Function],
    "name": "image",
    "parseHTML": [Function],
    "renderHTML": [Function],
  },
  "name": "image",
  "options": {
    "HTMLAttributes": {},
    "allowBase64": false,
    "inline": false,
  },
  "parent": null,
  "storage": {},
  "type": "node",
}
`;
